modules/ROOT/pages/_partials/commons/common-indexing.adoc

```diff
@@ -109,6 +109,183 @@ include::{root-partials}block_tabbed_code_example.adoc[]
 
 endif::is-c[]
 
+== Array Indexing 
 
+Couchbase Lite 3.2.1 introduces functionality to optimize querying arrays.
+Array `UNNEST` to unpack arrays within a document to allow joins with the parent object, and array indexes for indexing nested array properties and allow more efficient queries with `UNNEST`. 
 
+=== Array UNNEST
```

Comment:
IMO, the UNNEST and UNNEST Syntax section should be described as another type of the SQL++ expression, not at the index section. 

https://preview.docs-test.couchbase.com/CBL321/couchbase-lite/current/swift/query-n1ql-mobile.html#lbl-literals

From there, we can have a link to the Array Index section here.

Reply:
Moved!